lib/rubocop/cop/mixin/parentheses.rb

```diff
@@ -15,8 +15,20 @@ def autocorrect(node)
         lambda do |corrector|
           corrector.remove(node.loc.begin)
           corrector.remove(node.loc.end)
+
+          if ternary_operation?(node) && next_char_is_question_mark?(node)
+            corrector.insert_after(node.loc.end, ' ')
+          end
         end
       end
+
+      def ternary_operation?(node)
```

Comment:
This checks if `node` is the ternary *condition*, I think. 🙂

Reply:
Don't we already have some util or extension for this?